Content.Server/EntityEffects/Effects/ChemEyeDamageChange.cs

```diff
@@ -2,23 +2,25 @@
 using Content.Shared.Eye.Blinding.Systems;
 using JetBrains.Annotations;
 using Robust.Shared.Prototypes;
+using System.Text.Json.Serialization;
 
 namespace Content.Server.EntityEffects.Effects;
 
 /// <summary>
 /// Heal or apply eye damage
 /// </summary>
 [UsedImplicitly]
-public sealed partial class ChemHealEyeDamage : EntityEffect
+public sealed partial class ChemEyeDamageChange : EntityEffect
 {
     /// <summary>
     /// How much eye damage to add.
     /// </summary>
     [DataField]
+    [JsonPropertyName("amount")]
     public int Amount = -1;
 
     protected override string? ReagentEffectGuidebookText(IPrototypeManager prototype, IEntitySystemManager entSys)
-        => Loc.GetString("reagent-effect-guidebook-cure-eye-damage", ("chance", Probability), ("deltasign", MathF.Sign(Amount)));
+        => Loc.GetString("reagent-effect-guidebook-eye-damage-change", ("chance", Probability), ("deltasign", MathF.Sign(Amount)));
```

Comment:
```suggestion
        => Loc.GetString("reagent-effect-guidebook-change-eye-damage", ("chance", Probability), ("deltasign", MathF.Sign(Amount)));
```
Our naming convention usually has things in this order